### PR TITLE
aws-s3-website: add page

### PR DESCRIPTION
**Pull request by Isaac-Carrascal**
Add an initial description of the local options for aws s3 website command.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** j

### CODE REVIEW

File: pages/common/aws-s3-website.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+> Set the website configuration for a bucket.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/s3/website.html>.
+
+- Configure a bucket as a static website:
+
+`aws s3 website s3://bucket-name/ --index-document index.html`
+
+- Configure an error page for the website:
+
+`aws s3 website s3://bucket-name/ --index-document index.html --error-document error.html`
```
Comment (kbdharun) [inline pages/common/aws-s3-website.md:12]: I have a minor suggestion for this page, we can wrap the placeholders inside `{{}}` to be highlighted in a different colour/style by our clients.

```suggestion
`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}}`

- Configure an error page for the website:

`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}} --error-document {{error.html}}`
```

File: pages/common/aws-s3-website.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# aws-s3-website
+
+> Set the website configuration for a bucket.
```
Comment (kbdharun) [inline pages/common/aws-s3-website.md:3]:
```suggestion
> Set the website configuration for a bucket.
> See also: `aws s3`.
```
I would suggest linking back to the main page.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Comment by Isaac-Carrascal** [comment]
> Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

Thanks for your suggestions. Looking forward to continue contributing to this great project.

**Comment by kbdharun** [comment]
@Isaac-Carrascal Was this PR accidentally closed?

File: pages/common/aws-s3-website.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+> See also: `aws s3`.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/s3/website.html>.
+
+- Configure a bucket as a static website:
+
+`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}}`
+
+- Configure an error page for the website:
+
+`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}} --error-document {{error.html}}`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/aws-s3-website.md:13]: In this case, I am pretty sure it doesn't require path, will manually check it and get back.

**Pull request merged**